layouts/partials/components/component-title.html

```diff
@@ -0,0 +1,8 @@
+<div class="row text-center">
```

Comment:
Why not reuse https://github.com/felleslosninger/digdir-hugo-theme/blob/main/layouts/partials/components/component-title.html ?

Reply:
I'm sorry, I don't have the component-title.html in my copy of theme. See the image below for more info. I also tried to init and update the submodule, but it still didn't appear.

![image](https://user-images.githubusercontent.com/50879748/216436193-1258ed00-664d-435d-afc6-8185b94e6030.png)

Reply:
You need to pull the submodule. 
<img width="265" alt="image" src="https://user-images.githubusercontent.com/6088624/218695626-b2ded96b-0dee-4823-b6c1-f077fde4418d.png">

Reply:
And then push the update 
<img width="386" alt="image" src="https://user-images.githubusercontent.com/6088624/218697371-94da20d6-f2d1-45b5-b7b5-25830457ce45.png">

Reply:
OK! Thanks for the tips! I'll try it now.

Reply:
The command below did the trick:
`git submodule update --recursive --remote`